messaging-modules/apache-camel/src/main/java/com/baeldung/camel/apache/file/FileProcessor.java

```diff
@@ -5,14 +5,16 @@
 
 import org.apache.camel.Exchange;
 import org.apache.camel.Processor;
+import org.springframework.stereotype.Component;
 
+@Component
 public class FileProcessor implements Processor {
 
     public void process(Exchange exchange) throws Exception {
```

Comment:
Sorry one thing I missed, can we add an `@Override` here to clarify it is the interface method implementation

Reply:
Done